.github/workflows/build.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v4
       with:
-        go-version: "^1.21"
```

Comment:
Not sure if you are aware but you can use `stable` here to automatically use the latest release. It should be pretty safe due to Go's [compatibility guarantee](https://go.dev/blog/compat).

Reply:
we haven't done that, so that we can have a unified strategy - and move all the services to 1.22 at the same time.